flavors/standard-EXASOL-7.1.0-without-python2.7/flavor_base/build_deps/packages/apt_get_packages

```diff
@@ -0,0 +1,11 @@
+software-properties-common
```

Comment:
Here we should create a ticket to pin this dependencies in the future.

Reply:
Created #292